minard/views.py

```diff
@@ -1296,6 +1318,33 @@ def pca_run_detail(run_number):
                            run_number=run_number,
                            run=run)
 
+@app.route('/burst_run_detail/<run_number><subrun><sub>')
```

Comment:
I think it would be nice to separate these with a slash:

```python
@app.route('/burst_run_detail/<run>/<subrun>/<sub>')
```

Even better, you can have flask automatically turn them into integers by doing:

```python
@app.route('/burst_run_detail/<int:run>/<int:subrun>/<int:sub>')
```